PGM/src/main/java/tc/oc/pgm/polls/commands/PollCommands.java

```diff
@@ -0,0 +1,89 @@
+package tc.oc.pgm.polls.commands;
+
+import com.sk89q.minecraft.util.commands.Command;
+import com.sk89q.minecraft.util.commands.CommandContext;
+import com.sk89q.minecraft.util.commands.CommandException;
+import com.sk89q.minecraft.util.commands.CommandPermissions;
+import com.sk89q.minecraft.util.commands.NestedCommand;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import tc.oc.api.docs.PlayerId;
+import tc.oc.commons.bukkit.chat.PlayerComponent;
+import tc.oc.commons.bukkit.nick.IdentityProvider;
+import tc.oc.commons.core.chat.Audiences;
+import tc.oc.commons.core.chat.Component;
+import tc.oc.commons.core.commands.Commands;
+import tc.oc.commons.core.commands.TranslatableCommandException;
+import tc.oc.pgm.commands.CommandUtils;
+import tc.oc.pgm.polls.Poll;
+import tc.oc.pgm.polls.PollEndReason;
+import tc.oc.pgm.polls.PollManager;
+
+import javax.inject.Inject;
+
+public class PollCommands implements Commands {
+
+    private final PollManager pollManager;
+    private final IdentityProvider identityProvider;
+    private final Audiences audiences;
+
+    @Inject PollCommands(PollManager pollManager, IdentityProvider identityProvider, Audiences audiences){
+        this.pollManager = pollManager;
+        this.identityProvider = identityProvider;
+        this.audiences = audiences;
+    }
+
+    @Command(
+        aliases = {"poll"},
+        desc = "Poll commands",
+        min = 1,
+        max = -1
+    )
+    @NestedCommand({PollSubCommands.class})
+    public void pollCommand() {
+    }
+
+    @Command(
+        aliases = {"vote"},
+        desc = "Vote in a running poll.",
+        usage = "[yes|no]",
+        min = 1,
+        max = 1
+    )
+    @CommandPermissions("poll.vote")
+    public void vote(CommandContext args, CommandSender sender) throws CommandException {
+        PlayerId voter = CommandUtils.senderToMatchPlayer(sender).getPlayerId();
+        Poll currentPoll = pollManager.getPoll();
+        if(currentPoll != null) {
+            if(args.getString(0).equalsIgnoreCase("yes")) {
```

Comment:
Maybe have a cleaner way to deal with values. Perhaps use an enum to hold the possible values of a vote.

Reply:
I think the check is fine. Polls, at least for now, are either yes or no.